File: src/model/Visita.kt
package model

class Visita(val cidadao: Cidadao, val agenteSaude: AgenteSaude, var dataVisita:String = "", var descricao:String = "") {
    fun exibirVisita(){
        println(cidadao.nome)
        println(cidadao.cpf)
        println(agenteSaude.nome)
        println(agenteSaude.area)
        println(descricao)
        println(dataVisita)
    }
}

File: lib/models/Visita.dart
import 'package:sistema_saude/models/AgenteSaude.dart';
import 'package:sistema_saude/models/Cidadao.dart';

class Visita{
  Cidadao cidadao;
  Agentesaude agente;
  String dataVisita;
  String observacao;

  Visita(this.cidadao, this.agente, this.dataVisita, this.observacao);
}